src/renderers/webgl/WebGLBackground.js

```diff
@@ -68,7 +68,7 @@ function WebGLBackground( renderer, cubemaps, cubeuvmaps, state, objects, alpha,
 					new BoxGeometry( 1, 1, 1 ),
 					new ShaderMaterial( {
 						name: 'BackgroundCubeMaterial',
-						uniforms: cloneUniforms( ShaderLib.cube.uniforms ),
+						uniforms: cloneUniforms( ShaderLib.cube.uniforms ), // todo: use background shader, instead
```

Comment:
@Mugen87 suggested that perhaps the background shader can be modified to support blurry backgrounds.